Anime-App/Feature/Anime/View/AnimeViewController.swift

```diff
@@ -114,13 +121,15 @@ extension AnimeViewController {
         ])
     }
     
+    
     func getAnime() {
-        
+        isLoading = true
         self.spinner.startAnimating()
         animeViewModel.getAnime { [weak self] res in
             switch res {
             case .success:
                 self?.tableView.reloadData()
+                self?.isLoading = false
```

Comment:
Do you even need this isLoading? What is it even doing? It doesn't look like you use it for anything. Again Tobi this is messy and lazy. You've really got to cut this out. This is the type of stuff that will frustrate people when they review your work.